packages/ic-response-verification/tests/v2_response_verification_certification_scenarios.rs

```diff
@@ -296,13 +296,12 @@ mod tests {
 
```

Comment:
Using the CEL builder in our integration tests means we have a lot less handwritten CEL expressions. Eventually I'd like to get rid of all of them except for the ones that are used to test the CEL parser and builder.

There's still a few more around, but refactoring the tests wasn't my focus. I ran into issues in these tests because of changes that are coming in another PR. The Certification object from ic-http-certification is using references now, so after integrating with that the CEL parser functions have some lifetime restrictions on their parameters that makes them difficult to work with in certain situations and it was much easier to just use the CEL builder in the tests.